test/TestProjects/ExtensionClientName/Generated/AutoRestParameterFlatteningRestClient.cs

```diff
@@ -57,7 +57,7 @@ internal HttpMessage CreateRenamedOperationRequest(string renamedPathParameter,
         /// <param name="renamedQueryParameter"> The String to use. </param>
         /// <param name="renamedBodyParameter"> The RenamedSchema to use. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="renamedPathParameter"/>, <paramref name="renamedQueryParameter"/>, or <paramref name="renamedBodyParameter"/> is null. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="renamedPathParameter"/>, <paramref name="renamedQueryParameter"/> or <paramref name="renamedBodyParameter"/> is null. </exception>
```

Comment:
It looks like we are validating query and body parameter below.  This is because it's the autorest client?

Reply:
Yes